meta0v2/meta0_client.c

```diff
@@ -28,7 +28,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "meta0_utils.h"
 
 static addr_info_t addr = {{0}};
-static gchar *namespace = NULL;
+static gchar namespace[LIMIT_LENGTH_NSNAME] = {0};
```

Comment:
Constants like `LIMIT_LENGTH_NSNAME` should be clarified to tell if they include the final `'\0'` character or not. `metautils_message_extract_url` supposes they do not include it.